.github/PULL_REQUEST_TEMPLATE

```diff
@@ -2,21 +2,42 @@
 
 Add a few words about why this PR is needed and what is its scope.
 
+Add any comments about trade-offs (if any) made in this PR and the reasoning behind them.
+
 Add mentions of things that are not covered here and are planed to be done
 in separate PRs.
 
 
-## Remove this section
-
-Please have a look at [our developer documentation](https://twistedmatrix.com/trac/wiki/TwistedDevelopment#SubmittingaPatch) before submitting your Pull Request.
-
-Please note that the trac ticket, news fragment, and review submission portions of this process apply to *all* pull requests, no matter how small; if you don't do them, it's likely that nobody will even notice your PR needs a review.
-
-
 ## Contributor Checklist:
 
 * [ ] The associated ticket in Trac is here: https://twistedmatrix.com/trac/ticket/<!-- Create a new one at https://twistedmatrix.com/trac/newticket and replace this comment with the ticket number. -->
-* [ ] I ran `tox -e lint` to format my patch to meet the [Twisted Coding Standard](https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html)
 * [ ] I have created a newsfragment in src/twisted/newsfragments/ (see: [News files](https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles))
-* [ ] I have updated the automated tests.
+* [ ] The title of the PR starts with the associated Trac ticket number (without the `#` character).
+
+* [ ] I have updated the automated tests and checked that all checks for the PR are green.
+
 * [ ] I have submitted the associated Trac ticket for review by adding the word `review` to the keywords field in Trac, and putting a link to this PR in the comment; it shows up in https://twisted.reviews/ now.
+* [ ] I have added `twisted/twisted-contributors` teams to the PR `Reviewers`.
+
+* [ ] The merge commit will use the below format
+  The first line is automatically generated by GitHub based on PR ID and branch name.
+  The other lines generated by GitHub should be replaced.
+
+```
+Merge pull request #123 from twisted/4356-branch-name-with-trac-id
+
+Author: <github_username>, <github_usernames_if_more_authors>
+Reviewer: <github_username>, <github_usernames_if_more_reviewers>
+Fixes: ticket:<trac_ticket_number>, ticket:<another_if_more_in_one_PR>
+
+Long description providing a summary of these changes.
+(as long as you wish)
+```
+
+
+## Remove this section
+
+Have a look at [our developer documentation](https://twistedmatrix.com/trac/wiki/TwistedDevelopment#SubmittingaPatch) before submitting your Pull Request.
+
+Note that the Trac ticket, news fragment, and review submission portions of this process apply to *all* pull requests, no matter how small;
+if you don't do them, it's likely that nobody will even notice your PR needs a review.
```

Comment:
Some projects put this sort of stuff in a comment so it doesn't have to be deleted to not spam everyone looking at the issue.  Regardless, it seems like this warning should go at the beginning very beginning.  It is exactly for people that don't follow the checklist, not for people that do carefully follow the checklist and read every detail down to line 43.